### ISSUE TITLE
Detect SQLite function sqlite_compileoption_get

### ISSUE THREAD

**Issue by RedXanadu** [coreruleset/coreruleset#2717]
## Description

Relates to **I3LZGU1H**

Associated PR: #2718

---

We do not detect the SQLite function `sqlite_compileoption_get` which means payloads like this are not detected:

```
curl -v -o/dev/null 'localhost' --data $'email=admin%40juice-sh.op\'and%20not%20sqlite_compileoption_get%20(id)--'
```

We already have `sqlite_compileoption_used` added to rule 942151, so let's also add `sqlite_compileoption_get` and an associated test.

Note: I checked and couldn't find any other SQLite functions of the form `sqlite_compileoption_*` to consider.

**Comment by RedXanadu** [comment]
PR merged. Closing issue.
